doc/md/fundamentals/1-basic-syntax/9-operators.md

```diff
@@ -16,7 +16,7 @@ Motoko provides various operators for working with numbers, text, and boolean va
 
```

Comment:
I can't comment on the line above, but
```
| Bitwise    | Operations on individual bits   | `&`, `|`, ^`, `<<`, `>>`, `<<>`, `<>>` |
```
renders badly, maybe you need to escape bitwise or (vertical bar)
```
| Bitwise    | Operations on individual bits   | `&`, `\|`, ^`, `<<`, `>>`, `<<>`, `<>>` |
```

Reply:
(there's one or two more in the file - search for `|`)

Reply:
Using `\|` does not work (at least in my doc preview). you literally need to write  <code>&#124;</code> (as I suggested above).